src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -47,7 +47,7 @@ public class FileNameBuilder : IBuildFileNames
         private readonly ICached<bool> _patternHasEpisodeIdentifierCache;
         private readonly Logger _logger;
 
-        private static readonly Regex TitleRegex = new Regex(@"(?<escaped>\{\{|\}\})|\{(?<prefix>[- ._\[(]*)(?<token>(?:[a-z0-9]+)(?:(?<separator>[- ._]+)(?:[a-z0-9]+))?)(?::(?<customFormat>[a-z0-9+-]+(?<!-)))?(?<suffix>[- ._)\]]*)\}",
+        private static readonly Regex TitleRegex = new Regex(@"(?<escaped>\{\{|\}\})|\{(?<prefix>[- ._\[(]*)(?<token>(?:[a-z0-9]+)(?:(?<separator>[- ._]+)(?:[a-z0-9]+))?)(?::(?:[ ]+)?(?<customFormat>[ a-z0-9+-]+(?<!-| )))?(?<suffix>[- ._)\]]*)\}",
```

Comment:
Suggestion is more inline with what we do elsewhere with dashes and spaces and removed what appears to be an unnecessary space match

```suggestion
        private static readonly Regex TitleRegex = new Regex(@"(?<escaped>\{\{|\}\})|\{(?<prefix>[- ._\[(]*)(?<token>(?:[a-z0-9]+)(?:(?<separator>[- ._]+)(?:[a-z0-9]+))?)(?::(?<customFormat>[ a-z0-9+-]+(?<![- ])))?(?<suffix>[- ._)\]]*)\}",
```

Reply:
Applied your suggestion. The leading space match was to allow for `{Custom Format: FORMATNAME}` with a leading space, but that seems unnecessary unless someone requests it